### ISSUE TITLE
Different results with AlexNet on SHHB

### ISSUE THREAD

**Issue by MagazzuGaetano** [gjy3035/C-3-Framework#82]
I was trying to reproduce the results reported in the repo for the alexnet experiment on the SHHB dataset.

I trained alexnet using the recommended parameters in "results_reports/AlexNet/SHHB/config.py" 
"results_reports/AlexNet/SHHB/setting.py"
and stopped at 400 epochs instead of 406 epochs which is reported in the
"results_reports/AlexNet/SHHB/readme.md",

I initialized the weights of the network with those of pre-trained on ImageNet, but the results do not seem to be correct.

###########################
results obtained: mse: 20.41, mae: 13.22 (validated on the test set)
repo results: mse: 21.7, mae: 13.6 
###########################

In addition to the final [mse / mae] results which are different, 
I note that the first iterations already have different [mse / mae / loss] values,
other than those reported in the github repo

I was wondering if in the experiment reported in the repo has been considered the following observations:
- The RandomCrop in "./datasets/SHHB/loading_data.py"?
- The use of a pre-trained model on GCC instead of on ImageNet?

I report the log of my experiment:
[Log_exp1.txt] (https://github.com/gjy3035/C-3-Framework/files/4667995/log_exp1.txt)

**Comment by gjy3035** [comment]
1. Slight difference is normal
2. ImageNet

**Comment by MagazzuGaetano** [comment]
Is it normal that in the last epochs there are such fluctuations between the values?

**Comment by MagazzuGaetano** [comment]
thanks
